waspc/cli/Main.hs

```diff
@@ -31,6 +31,7 @@ main = do
         ["version"] -> Command.Call.Version
         ["build"] -> Command.Call.Build
         ["telemetry"] -> Command.Call.Telemetry
+        ("commands" : keyword) -> Command.Call.Commands keyword
```

Comment:
Maybe make it `["commands", keyword]` just to be consistent with the code above?

Reply:
Also, let's rename `keyword` to `subcommand`?